### PR TITLE
Add post: Adding a navigation bar

### PR DESCRIPTION
**Pull request by frankieroberto**

### CODE REVIEW

File: app/posts/apply-for-teacher-training/2023-06-12-adding-a-navigation-bar.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ogImage:
+  src: /apply-for-teacher-training/adding-a-navigation-bar/navigation-bar.png
+  alt: "Illustration showing 3 horizontal elements: a bar with a black background and a crown and GOV.UK in white, a bar with a white background and a blue 'BETA' badge, and a bar with a grey background and 'Your applications' and 'Sign out' links in blue text."
+---
+
+![Illustration showing 3 horizontal elements: a bar with a black background and a crown and GOV.UK in white, a bar with a white background and a blue 'BETA' badge, and a bar with a grey background and 'Your applications' and 'Sign out' links in blue text.](navigation-bar.png)
+
+We added a navigation bar to the candidate interface.
+
+Currently, this only contains 2 items, but we added the navigation bar to enable us to add additional navigation opens as we [change the application process](/apply-for-teacher-training/changing-application-process/) in future.
```
Comment (MaeveRoseveare) [inline app/posts/apply-for-teacher-training/2023-06-12-adding-a-navigation-bar.md]: Is there a typo with 'navigation opens' should that be 'navigation items'?

**Review by MaeveRoseveare — APPROVED** [review]
Looks great!

File: app/posts/apply-for-teacher-training/2023-06-12-adding-a-navigation-bar.md (hunk 1 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2023-06-12-adding-a-navigation-bar.md]: yes! 🤦

Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-06-12-adding-a-navigation-bar.md]:
```suggestion
Currently, this only contains 2 items, but we added the navigation bar to enable us to add additional navigation items as we [change the application process](/apply-for-teacher-training/changing-application-process/) in future.
```

**Pull request merged**